tiledb/sm/filter/filter_pipeline.cc

```diff
@@ -85,9 +87,7 @@ FilterPipeline::FilterPipeline(
   max_chunk_size_ = other.max_chunk_size_;
 }
 
-FilterPipeline::FilterPipeline(FilterPipeline&& other) {
-  swap(other);
```

Comment:
Same problem as before. The compiler looked through `swap` and saw uninitialized member access.